[2023-08-04T09:27:52.793+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: toronto_real_estate_etl.Extract manual__2023-08-04T09:27:51.764069+00:00 [queued]>
[2023-08-04T09:27:52.800+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: toronto_real_estate_etl.Extract manual__2023-08-04T09:27:51.764069+00:00 [queued]>
[2023-08-04T09:27:52.800+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-04T09:27:52.810+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): Extract> on 2023-08-04 09:27:51.764069+00:00
[2023-08-04T09:27:52.813+0000] {standard_task_runner.py:57} INFO - Started process 88 to run task
[2023-08-04T09:27:52.816+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'toronto_real_estate_etl', 'Extract', 'manual__2023-08-04T09:27:51.764069+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmprei8yyin']
[2023-08-04T09:27:52.817+0000] {standard_task_runner.py:85} INFO - Job 95: Subtask Extract
[2023-08-04T09:27:52.854+0000] {task_command.py:410} INFO - Running <TaskInstance: toronto_real_estate_etl.Extract manual__2023-08-04T09:27:51.764069+00:00 [running]> on host e5187b3c4256
[2023-08-04T09:27:52.912+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='toronto_real_estate_etl' AIRFLOW_CTX_TASK_ID='Extract' AIRFLOW_CTX_EXECUTION_DATE='2023-08-04T09:27:51.764069+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-04T09:27:51.764069+00:00'
[2023-08-04T09:27:52.913+0000] {Extract.py:31} INFO - https://www.kijiji.ca/b-apartments-condos/page-1/c37l1700273
[2023-08-04T09:28:05.671+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/src/main.py", line 22, in extract
    e.getListings(2,date)
  File "/opt/airflow/dags/src/Extract.py", line 44, in getListings
    s.getHeaderDetails()
  File "/opt/airflow/dags/src/Scrape.py", line 52, in getHeaderDetails
    self.soup = BeautifulSoup(self.r.content, 'html.parser')
AttributeError: 'Scrape' object has no attribute 'r'
[2023-08-04T09:28:05.681+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=toronto_real_estate_etl, task_id=Extract, execution_date=20230804T092751, start_date=20230804T092752, end_date=20230804T092805
[2023-08-04T09:28:05.696+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 95 for task Extract ('Scrape' object has no attribute 'r'; 88)
[2023-08-04T09:28:05.740+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-08-04T09:28:05.756+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
